src/tests/uat/live_response_api_uat.py

```diff
@@ -89,23 +91,9 @@ def flatten_processes_response_filter(response, process_name):
     return [item['process_pid'] for item in response if process_name in item['process_path']]
 
 
-def main():
-    """Script entry point"""
-    global ORG_KEY
-    global HOSTNAME
-    parser = build_cli_parser()
-    args = parser.parse_args()
-    print_detail = args.verbose
-
-    if print_detail:
-        print(f"args provided {args}")
-
-    cb = get_cb_cloud_object(args)
-    HEADERS['X-Auth-Token'] = cb.credentials.token
-    ORG_KEY = cb.credentials.org_key
-    HOSTNAME = cb.credentials.url
-
-    print(13 * ' ' + 'Live Response Commands')
+def synchronous_version(cb):
```

Comment:
I would split this into two separate UAT tests as this is a long file now and is hard to read and understand

Reply:
Done.